docs/modules/development/pages/rest/config_management.adoc

```diff
@@ -0,0 +1,62 @@
+= Config Management
+
+It is the generic specification of Config Management RESTful API
```

Comment:
Could we update this to be a bit more specific on why the user uses this? For example:

Use the Confg Management API to ...

Reply:
extra comment added